packages/api/src/command/medical/patient-discovery-result/get-patient-discovery-result.ts

```diff
@@ -0,0 +1,13 @@
+import { PatientDiscoveryResult } from "../../../domain/medical/patient-discovery-result";
+import { PatientDiscoveryResultModel } from "../../../models/medical/patient-discovery-result";
+
+export async function getPatientDiscoveryResults(
+  requestId: string
+): Promise<PatientDiscoveryResult[]> {
+  return await PatientDiscoveryResultModel.findAll({
+    where: {
+      requestId,
+      status: "success",
+    },
+  });
+}
```

Comment:
Wondering if it would be beneficial to also receive failures that way we can log them or send them to sentry

Reply:
Isn't this the query on the table you created, @Orta21 ? If so, we need to add an index to the table, right?

Reply:
@RamilGaripov whats the final verdict for this?